pkg/catalog/ws_http.go

```diff
@@ -150,7 +70,7 @@ func (mc *MeshCatalog) witesandHttpClient() {
 		}
 		// learn remote pods
 		for clusterId, remoteK8s := range wc.ListRemoteK8s() {
-			remotePods, err := queryAllPodRemote(remoteK8s.OsmIP)
+			remotePods, err := mc.QueryAllPodRemote(wc, remoteK8s.OsmIP)
```

Comment:
I am not sure I understand why we need both QueryRemoteOSM() and QueryAllPodRemote() ?
What is the difference ?